swagger/public_v1/component_schemas/CourseAttributes.yml

```diff
@@ -0,0 +1,315 @@
+    type: string
+    format: date
+    description: "Date from which applications can be submitted."
+    example: "2019-10-08"
+  bursary_amount:
+    type: integer
+    description: "Bursary amount for this course."
+    example: 9000
+  bursary_requirements:
+    type: string
+    description: "Description of requirements to be eligible for a bursary."
+    example: "a degree of 2:2 or above in any subject"
+  changed_at:
+    type: string
+    format: date-time
+    description: |
+      Date-time timestamp of when this course or any of it's related data changed.
+    example: "2019-06-13T10:44:31Z"
+  code:
+    type: string
+    description: |
+      Code that uniquely identifies this course within it's providers list of courses.
+    maxLength: 4
+    minLength: 4
+    example: 3GTY
+  course_length:
+    type: string
+    description: |
+      Text describing how long the course runs.
+    example: "OneYear"
+  created_at:
+    type: string
+    format: date-time
+    description: "Timestamp of when this course was created."
+    example: "2019-06-13T10:44:31Z"
+  fee_details:
+    type: string
+    format: markdown
+    description: "Further details about the fees for this course, if applicable."
+    example: fee_details
+  fee_international:
+    type: integer
+    description: "Fee for international students (optional)."
+    example: 13000
+  fee_domestic:
+    type: integer
+    description: "Fee for UK and EU students."
+    example: 9200
+  financial_support:
+    type: string
+    format: markdown
+    description: "Details about financial support offered, if any."
+    example: financial_support
+  findable:
+    type: boolean
+    description: |
+      Is this course currently visible on the Find Postgraduate Teacher Training service.
+    example: true
+  funding_type:
+    type: string
+    description: |
+      The type of funding that maybe provided to candidates, if any.
+    example: apprenticeship
+    enum:
+      - salary
+      - apprenticeship
+      - fee
+  gcse_subjects_required:
+    type: string
+    example: [ maths, english ]
+    enum:
+      - []
+      - [maths, english]
+      - [maths, english, science]
+    description: "GSCE standard equivalent required for this level of course."
+  has_bursary:
+    type: boolean
+    description: "Are any bursaries available for this course?"
+    example: true
+  has_early_career_payments:
+    type: boolean
+    description: "Are early career payments available for this course?"
+    example: true
+  has_scholarship:
+    type: boolean
+    description: "Are scholarships available for this course?"
+    example: true
+  has_vacancies:
+    type: boolean
+    description: "Do any of the locations for this course have vacancies?"
+    example: true
+  how_school_placements_work:
+    type: string
+    format: markdown
+    description: |
+      Additional information about the schools applicants will be teaching in.
+    example: how_school_placements_work
```

Comment:
Missing example

Reply:
Move to #1285

Reply:
Addressed here: https://github.com/DFE-Digital/teacher-training-api/pull/1285/files#diff-55ef93162c9cf3f0abd50e45926cc156R221-R224